docs/maya/ui_overview.md

```diff
@@ -201,17 +203,38 @@ Option boxes can be used to launch a UI to assign the name to the deformer and c
 - **Draw Fibers**. Enables *Muscle Fibers* Debugging from all the AdnRibbonMuscle and AdnMuscle in the scene.
 - **Hide Fibers**. Disables *Muscle Fibers* Debugging from all the AdnRibbonMuscle and AdnMuscle in the scene.
 
+#### Glue
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Edit Muscle](images/adonis_menu_glue.png)
+  <figcaption><b>Figure 16</b>: AdonisFX Menu Edit Glue section.</figcaption>
+</figure>
+
+- **Add Inputs**. Adds input meshes to the selected AdnGlue. The glue output mesh must be the last element in the selection.
+- **Remove Inputs**. Removes input meshes from the selected AdnGlue. The glue output mesh must be the last element in the selection. If only the glue geometry is selected, then all inputs will be removed.
+
 #### Debug
 
 <figure style="width: 50%;" markdown>
   ![AdonisFX Menu Debug Section](images/adonis_menu_debug.png)
-  <figcaption><b>Figure 16</b>: AdonisFX Menu Debug section.</figcaption>
+  <figcaption><b>Figure 17</b>: AdonisFX Menu Debug section.</figcaption>
 </figure>
 
 - **Refresh Debugger**. Refreshes the scene graph for debugging purposes by making sure the AdnDataNode and the AdnDebugLocator exist in the scene and are properly connected.
 
 ### Tools section
 
+#### Utils
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Debug Section](images/adonis_menu_utils.png)
+  <figcaption><b>Figure 18</b>: AdonisFX Menu Utils section.</figcaption>
+</figure>
+
+- **Upgrade Deprecated Nodes**. Refreshes the node graph and upgrades all deprecated nodes making sure that the scene still works the same. The deprecated nodes are: AdnWeightsDisplayNode (v1.4.0).
```

Comment:
Now that I see this, I am not that sure with exposing the deprecated node here as it will be there for ever.
```suggestion
- **Upgrade Deprecated Nodes**. Refreshes the node graph and upgrades all deprecated nodes making sure that the scene still works the same.
```

Reply:
Removed.